racetime/static/racetime/style/race.css

```diff
@@ -101,23 +113,48 @@ aside, main {
     position: relative;
 }
 
-.race-chat > .disconnected {
+.race-chat>.disconnected {
     cursor: default;
     display: none;
     font-weight: bold;
     justify-content: center;
     line-height: 27px;
     position: absolute;
-        top: 10px;
-        right: 0;
-        left: 0;
+    top: 10px;
+    right: 0;
+    left: 0;
 }
 
-.race-chat.disconnected > .disconnected {
+.race-chat>.scrollwarning {
+    cursor: default;
```

Comment:
I think `cursor: pointer` is a better idea since this is a clickable element.

Reply:
Makes sense I just pulled the existing CSS you had.